test/src/utils/transition-manager.spec.js

```diff
@@ -395,7 +397,7 @@ test('TransitionManager#TRANSITION_EVENTS', t => {
       // testing easing function
       let testEasingFunc = testCase.input[ti].transitionEasing;
       if (mode === TRANSITION_EVENTS.UPDATE) {
-        const completion = (time[1] - time[0]) / testCase.input[ti].transitionDuration;
+        const completion = 100 / testCase.input[ti].transitionDuration;
```

Comment:
Where did you get this number `100`?

Reply:
https://github.com/uber/react-map-gl/pull/798/files#diff-a02c4dd2c586d5f96eeeb6581028fb59R376